### DISCUSSION TITLE
https://kumaarbalbir.github.io/Roll-your-dice/

### DISCUSSION THREAD

**Discussion by byteB8** [community/community#23970] [Other Feature Feedback, Questions, & Ideas]
<p><a href="https://kumaarbalbir.github.io/Roll-your-dice/" class="onebox" target="_blank" rel="noopener nofollow ugc">https://kumaarbalbir.github.io/Roll-your-dice/</a><br>
why my github page is only showing repo name? not loading the data?</p>

**Comment by canuckjacq** [comment, marked as answer]
<p>:wave: Welcome!</p>
<p>There are a couple of things.</p>
<p>First, Pages will look for an <code>index.html</code>, <code>index.md</code> or <code>readme.md</code> page to display. You only have one of those (<code>readme.md</code>), and I suspect that’s not the file you’re expecting to see.</p>
<p>If <a href="https://kumaarbalbir.github.io/Roll-your-dice/dicee.html">this</a> is the page you want to see, you’ll want to rename it to <code>index.html</code>.</p>
<p>Additionally, you will want to check the html file contents - I can see that in many cases you’re referring to directories that don’t exist in your repository, so those assets won’t display properly.</p>

**Comment by byteB8** [comment]
<p>Thanks a lot!! It is now working properly</p>
